src/vm/contexts.rs

```diff
@@ -223,7 +223,7 @@ pub struct ContractContext {
     pub meta_ft: HashMap<ClarityName, FungibleTokenMetadata>,
     pub data_size: u64,
     /// track the clarity version of the contract
-    clarity_version: ClarityVersion,
+    pub clarity_version: ClarityVersion,
```

Comment:
Note: Making this public! What do you think @kantai ?

Reply:
I'd prefer if consumers just used `.get_clarity_version()` -- this clarity_version instance variable should be immutable, and making it private helps ensure that's the case.

Reply:
oh.. :/... I guess this wasn't even needed in this PR. Must have been copied from another PR. However, in future PR's I'll de-pub this line and use `get_clarity_version`.